packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -174,6 +183,21 @@ function Field({
     updateSelectText(newList);
   }
 
+  const handlePaste = (event: ClipboardEvent, index: number) => {
+    const paste = event.clipboardData.getData("text");
+    const formattedValues = paste
+      .split(/[\n,;]+/)
+      .map((value) => ({ placeholder: "New Option", value: value.trim(), id: uuidv4() }))
```

Comment:
I don't think we need to provide a placeholder when value is already there. We are anyway going to reject all entries that don't have value.